gdx-freetype-jtransc/src/com/badlogic/gdx/graphics/g2d/freetype/FreeType.java

```diff
@@ -0,0 +1,710 @@
+			throw new RuntimeException("FreeType.GlyphSlot.getBitmapTop(): Not implemented.");
+		}
+
+		public boolean renderGlyph(int renderMode) {
+			return true;
+		}
+
+		public Glyph getGlyph() {
+			return new Glyph(this);
+		}
+	}
+
+	public static class Glyph {
+		private boolean rendered;
+		private GlyphSlot slot;
+		private Bitmap bitmap;
+
+		Glyph(GlyphSlot slot) {
+			this.slot = slot;
+		}
+
+		public void strokeBorder(Stroker stroker, boolean inside) {
+		}
+
+		public void toBitmap(int renderMode) {
+			if (bitmap == null) {
+				bitmap = slot.getBitmap();
+			}
+			rendered = true;
+		}
+
+		public Bitmap getBitmap() {
+			if (bitmap == null) {
+				bitmap = slot.getBitmap();
+			}
+			return bitmap;
+		}
+
+		public int getLeft() {
+			return toInt(slot.getMetrics().getHoriBearingX());
+		}
+
+		public int getTop() {
+			return toInt(slot.getMetrics().getHoriBearingY());
+		}
+	}
+
+	@HaxeImports(
+		"import lime.graphics.Image;"
+	)
+	@HaxeAddMembers(
+		"var image: Image;"
+	)
+	public static class Bitmap {
+		private static Face face;
+
+		Bitmap(Face face) {
+			this.face = face;
+			init(face);
+		}
+
+		@HaxeMethodBody(
+			"image = p0.font.renderGlyph(p0.glyph, p0.pixelHeight);"
+		)
+		private void init(Face face) {
+		}
+
+		@HaxeMethodBody(
+			"return image != null ? image.height : 1;"
+		)
+		public int getRows() {
+			throw new RuntimeException("FreeType.Bitmap.getRows(): Not implemented.");
+		}
+
+		@HaxeMethodBody(
+			"return image != null ? image.width : 10;"
+		)
+		public int getWidth() {
+			throw new RuntimeException("FreeType.Bitmap.getWidth(): Not implemented.");
+		}
+
+		@HaxeMethodBody(
+			"return image != null ? (image.buffer.width * image.buffer.bitsPerPixel) : 10;"
+		)
+		public int getPitch() {
+			throw new RuntimeException("FreeType.Bitmap.getPitch(): Not implemented.");
+		}
+
+		public ByteBuffer getBuffer() {
+			return ByteBuffer.wrap(getData());
+		}
+
+		@HaxeMethodBody(
+			"if(image == null) " +
+				" 	return new JA_B(10);" +
+				"var bytes = image.data.toBytes();" +
+				"return new JA_B(bytes.length, bytes);"
+		)
+		private byte[] getData() {
+			return new byte[0];
+		}
+
+		public Pixmap getPixmap(Format format, Color color, float gamma) {
+			Pixmap pixmap = new Pixmap(getWidth(), getRows(), Format.RGBA8888);
+			ByteBuffer buffer = pixmap.getPixels();
+			byte[] src = getData();
+			byte[] dest = new byte[src.length * 4];
+
+			byte r = (byte) (color.r * 0xFF);
+			byte g = (byte) (color.g * 0xFF);
+			byte b = (byte) (color.b * 0xFF);
+			int a = ((int) (color.a * 0xFF)) & 0xFF;
+
+			for (int i = 0; i < dest.length; ) {
+				// Use the specified color for RGB, blend the FreeType bitmap with alpha.
+				float alpha = (src[i >> 2] & 0xff) / 255f;
+				alpha = (float) Math.pow(alpha, gamma); // Inverse gamma.
+				dest[i++] = r;
+				dest[i++] = g;
+				dest[i++] = b;
+				dest[i++] = (byte) (a * alpha);
+			}
+
+			BufferUtils.copy(dest, 0, buffer, dest.length);
```

Comment:
Not sure if it is the case here due to jtransc current implementation overhead. But usually is is faster to create an IntBuffer from a ByteBuffer, pack RGBA pixel into an int and just write once to memory per pixel. Also if that loop is larger than 512, maybe you can precalculate all alphas by that gamma. To just do a lookup later.
Again thats a possible future optimization and not important.